src/main/java/org/cprover/CProverString.java

```diff
@@ -388,4 +388,26 @@ public static StringBuffer insert(
         return CProver.nondetWithoutNullForNotModelled();
     }
 
+    /**
+     * Converts an array of characters to a string. This uses a loop and
+     * therefore in test-generation the {@code count} will be limited by the
+     * unwind parameter.
+     * This assumes {@code value} is not null, its length is greater or equal to
```

Comment:
`This assumes` is a bit ambiguous. It looks like a precondition to the function. Maybe `This constrains {@code value} to be not null...`?